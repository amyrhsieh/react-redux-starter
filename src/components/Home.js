import React from 'react';
import Image from 'Images/image.jpg';

import styles from 'Styles/_styles.scss';

const Home = () => (
  <div>
    <h2>Home</h2>
    <p>Thank you for checking out my demo React app! The purpose of this app is to serve as a starting point for any web app that uses React, Webpack, Redux, React Final Form, Jest/Enzyme tests, and all of the other packages used in this demo. Not everything is fleshed out, but the very basics are in place (e.g. a form using React Final Form; redux store, reducer and action creators; usage of Bootstrap styles) to give a taste of how each of the tools are used. Each feature can be fleshed out further, depending on the needs of the web app.<br />
    -AH
    </p>

    <div className="center">
      <img src={Image} />
    </div>
  </div>
);

export default Home;
