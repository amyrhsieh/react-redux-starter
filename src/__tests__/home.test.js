import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home.js';

describe('Home (Snapshot)', () => {
  it('Home component renders Home', () => {
    const component = renderer.create(<Home />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
