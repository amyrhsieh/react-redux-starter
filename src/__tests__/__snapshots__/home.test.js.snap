// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home (Snapshot) Home component renders Home 1`] = `
<div>
  Home
</div>
`;
